Sources/Nimble/Matchers/BeAKindOf.swift

```diff
@@ -1,14 +1,22 @@
 import Foundation
 
-#if _runtime(_ObjC)
```

Comment:
Removal of this is breaking Linux build since `NMBObjCMatcher` is not available on Linux. `extension NMBObjCMatcher` should be still surrounded by the `#if _runtime(_ObjC)` condition.